vms/vms/settings.py

```diff
@@ -129,10 +129,17 @@
 
 # Instead of sending out real email, during development the emails will be sent
 # to stdout, where from they can be inspected.
-if DEBUG:
-    EMAIL_HOST = config('EMAIL_HOST', default='localhost')
-    EMAIL_PORT = config('EMAIL_PORT', default=1025, cast=int)
-    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+# if DEBUG:
+#     EMAIL_HOST = config('EMAIL_HOST', default='localhost')
+#     EMAIL_PORT = config('EMAIL_PORT', default=1025, cast=int)
+#     EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'  
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_PASSWORD = 'your_password'  
+EMAIL_HOST_USER = 'your_email'  
+EMAIL_PORT = 587
+EMAIL_USE_TSL = True  
```

Comment:
I don't understand this change :thinking: 

It's supposed to be like that so we can test easily during local dev

Reply:
But I don't think while pushing the changes we usually keep the email addresses and the passwords. The one who clones it, needs to add their personal ones before running this. This is what I learnt in my Internship. If I am wrong, do correct me :)

Reply:
Sorry for the late response but could you elaborate on that? I do not get you.

Reply:
Sure. What we used to do like whenever we used to push the changes in authentication part regarding login or signups, we deleted the dummy email Id used. Because after the changes are merged this email ID and password wont be valid for the other person who tries to clone and run. So kept that field empty and whenever someone tries to run, s/he must first put in their id and password in order to try the emails sent.

Reply:
If we are not supposed to do that way here then do let me know if I am wrong :)

Reply:
Oh I see. I will try to explain.

So basically, first things first, for anything that depends on an external service, it's generally much more efficient to keep it configurable, and that's exactly what's done here. If you can imagine, if for an external service like mail, we keep it hardcoded it can lead to all sorts of abuse, that is why the `config()` function is used (from a 3rd party package) which basically loads the required settings from a `.env` file.